context/app/static/js/components/home-revision/AssayTypeBarChart/AssayTypeBarChart.jsx

```diff
@@ -0,0 +1,152 @@
+import React, { useContext, useMemo } from 'react';
+import { scaleLinear, scaleOrdinal, scaleBand } from '@visx/scale';
+import { BarStackHorizontal } from '@visx/shape';
+import { Group } from '@visx/group';
+import { AxisTop, AxisLeft } from '@visx/axis';
+import { withParentSize } from '@visx/responsive';
+
+import { AppContext } from 'js/components/Providers';
+import useSearchData from 'js/hooks/useSearchData';
+import { formatAssayData, addSumProperty, sortBySumAscending } from './utils';
+import { useChartPalette } from './hooks';
+
+const organTypesQuery = {
+  size: 0,
+  aggs: {
+    organ_types: { terms: { field: 'origin_sample.mapped_organ.keyword', size: 10000 } },
+  },
+};
+
+const assayTypesQuery = {
+  size: 0,
+  aggs: {
+    mapped_data_types: {
+      composite: {
+        sources: [
+          {
+            mapped_data_type: {
+              terms: {
+                field: 'mapped_data_types.keyword',
+              },
+            },
+          },
+          {
+            organ_type: {
+              terms: {
+                field: 'origin_sample.mapped_organ.keyword',
+              },
+            },
+          },
+        ],
+        size: 10000,
+      },
+    },
+  },
+};
+
+const getDataType = (d) => d.mapped_data_type;
```

Comment:
Can we declare this within the function?  Just feels a little out of place here since it is only used once.

Reply:
Sounds good, thanks.